src/views/ConsentContinuationOut.vue

```diff
@@ -147,6 +158,7 @@
               :offset="{ top: 120, bottom: 40 }"
             >
               <SbcFeeSummary
+                class="ml-4"
```

Comment:
What do the other local filings (AR, COA, COD, Correction) do here?

What does this look like on mobile (responsive / narrow screen)? Normally the Affix component handles this. You may need a special ml class that only takes affect on sm breakpoints or larger. Ref: https://vuetifyjs.com/en/styles/spacing/

Reply:
The problem was that I don't know why, but I added no gutters to v-row. Removing it fixed the layout and now is consistent with other local filings.